api/securitytest/gitleaks.go

```diff
@@ -61,6 +78,29 @@ func (gitleaksScan *SecTestScanInfo) prepareGitleaksVulns() {
 
 	huskyCIgitleaksResults := types.HuskyCISecurityTestOutput{}
 	gitleaksOutput := gitleaksScan.FinalOutput.(GitleaksOutput)
+
+	if gitleaksScan.GitleaksTimeout {
+		gitleaksVuln := types.HuskyCIVulnerability{}
+		gitleaksVuln.Language = "Generic"
+		gitleaksVuln.SecurityTool = "Gitleaks"
+		gitleaksVuln.Severity = "nosec"
+		gitleaksVuln.Details = "It looks like your project is too big and huskyCI was not able to run Gitleaks."
```

Comment:
Should we add some instructions on how to run a full Gitleaks analysis? What do you think?
Maybe this detail section could point to a page in the wiki with instructions.

Reply:
Great idea! Can you create an issue so we can work on this guide soon? We could add it into our documentation at [http://husky.ci](http://husky.ci), what do you think?

Reply:
I just created this [issue](https://github.com/globocom/huskyCI/issues/451) so we can work on this guide soon!